### DISCUSSION TITLE
有人导出onnx成功了吗？

### DISCUSSION THREAD

**Discussion by ZXisSpider** [fishaudio/Bert-VITS2#297] [Q&A]
我利用2.3版本基模训练好自己的模型后试图用export_onnx.py导出onnx模型失败，调用的是V220_novq_dev文件夹里面的export_onnx函数。（做了一些修改，删除了其中所有关于emo的部分，因为checkpoint似乎不包含emo部分）。
我的命令行在跳出一大堆warning后，最后会在export sdp.onnx的部分蹦出Floating Point Exception，然后就停止导出了。
想问问有人成功导出过onnx文件吗？导出模型还需要哪些步骤呢？

**Comment by ZXisSpider** [comment]
我在mac os的环境导出成功了。。。真是玄学

**Reply by DoiiarX** [reply]
成功过，但是没法infer

**Reply by GDbbq** [reply]
能导出，但是infer结果不对

**Reply by Sunnycl** [reply]
> 我在mac os的环境导出成功了。。。真是玄学

能正常推理吗,我导出生成的音频要么没声音要么全是噪声

**Reply by DoiiarX** [reply]
> > 我在mac os的环境导出成功了。。。真是玄学
> 
> 能正常推理吗,我导出生成的音频要么没声音要么全是噪声

估计是采样率之类

**Comment by ZXisSpider** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
